docs/src/main/asciidoc/security.adoc

```diff
@@ -3,161 +3,185 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Security Architecture and Guides
+= Quarkus Security overview
 
 include::./attributes.adoc[]
 
-Quarkus Security provides the architecture, multiple authentication and authorization mechanisms, and other tools for the developers to build a production-quality security for their Quarkus applications.
+Quarkus Security is a framework that provides the architecture, multiple authentication and authorization mechanisms, and other tools for you to build secure and production-quality Java applications.
 
-This document provides a brief overview of Quarkus Security and links to the individual guides.
+== Security architecture
 
-== Getting Started
+The `HttpAuthenticationMechanism` interface is the main entry mechanism for securing HTTP applications in Quarkus. 
 
-Please see the xref:security-getting-started.adoc[Getting Started With Security] guide for a quick walkthrough through Quarkus Security where you can learn how to use xref:security-basic-auth-concept.adoc[Basic HTTP Authentication] mechanism and `JPA Identity Provider` to create `SecurityIdentity` and authorize a secure access to the endpoint with `Role Based Access Control`.
+The Quarkus Security Manager uses `HttpAuthenticationMechanism` to extract the authentication credentials from the HTTP request and delegates them to `IdentityProvider` to convert the credentials to `SecurityIdentity`.
```

Comment:
Not sure about the `Manager` bit - we don't really have such an entity, only conceptually, `The Quarkus Security` alone should be enough ?

Reply:
Well, I see `Manager` was already there before :-)- but may be we can remove it, what do you think ?

Reply:
:+1:

Reply:
I was actually going to ask you about the Quarkus Security Manager and whether we needed to talk more about that entity. But you've answered my question! Thanks